packages/jumpstarter-driver-network/jumpstarter_driver_network/driver.py

```
@@ -235,3 +237,24 @@ async def connect(self):
         self.logger.debug("Connecting Echo")
         async with StapledObjectStream(tx, rx) as stream:
             yield stream
+
+
+class WebsocketNetwork(NetworkInterface, Driver):
+    '''
+    Handles websocket connections from a given url.
+    '''
+    url: str
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)